knowledge-base/angular-report-viewer-in-multiple-tabs.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Angular Report Viewer in Multiple Tabs
+description: Sometimes when you want to display the report viewers in different tabs only in the first tab the report is rendered.
```

Comment:
I would leave out the unnecessary words: "When displaying Angular report viewer in different tabs it renders in the first tab only " something like that